test/remote/gateways/remote_adyen_test.rb

```diff
@@ -45,6 +45,14 @@ def test_successful_purchase
     assert_equal '[capture-received]', response.message
   end
 
+  def test_successful_purchase_no_cvv
+    credit_card = @credit_card
+    credit_card.verification_value=nil
```

Comment:
This is again stupidly minor, but RuboCop or your equivalent should be telling you to put spaces on either side of the `=`.